### PR TITLE
Fix Rule ensure logrotate activated

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Add test scenarios for Rule ensure_logrotate_activated
- Update OVAL definition to:
  - Allow other arguments to logrotate command
  - Ensure that `/etc/logrotate.conf` is the last config file
  - Check for both `logrotate.conf` AND `cron.daily`
- Add bash remediation

### CODE REVIEW

File: shared/fixes/bash/ensure_logrotate_activated.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# platform = multi_platform_rhel
+
+# daily rotation is configured
+grep -q "^daily$" /etc/logrotate.conf || echo "daily" >> /etc/logrotate.conf
+
+# remove any line configuring weekly, monthly or yearly rotation
+sed -i -r "/^(weekly|monthly|yearly)$/d" /etc/logrotate.conf
```
Comment (mpreisler) [inline shared/fixes/bash/ensure_logrotate_activated.sh]: this is not adding logrotate to crontab

Comment (yuumasato) [reply shared/fixes/bash/ensure_logrotate_activated.sh]: Hmm, so just configuring `logrotate.conf` is not enough?

Comment (mpreisler) [reply shared/fixes/bash/ensure_logrotate_activated.sh]: I don't know but I thought you have to call `/sbin/logrotate` periodically for the config file to even be used.

Comment (yuumasato) [reply shared/fixes/bash/ensure_logrotate_activated.sh]: It makes sense, updated everything to account for `logrotate.conf` and `cron.daily`.

Comment (mpreisler) [reply shared/fixes/bash/ensure_logrotate_activated.sh]: Wait, did you verify that? Because I am not sure about what I said.

Comment (yuumasato) [reply shared/fixes/bash/ensure_logrotate_activated.sh]: Yes, `man logrotate` states the following
> Normally, logrotate is run as a daily cron job.

Comment (yuumasato) [reply shared/fixes/bash/ensure_logrotate_activated.sh]: Also from man page:
> hourly       Log files are rotated every hour. Note that usually logrotate is configured to be run by cron daily. You have to change this configuration and run logrotate hourly to be able to really rotate logs hourly.

Configuring frequency of logrotation in `logrotate.conf` doesn't account for frequency of `logrotate` runs. The cron job also needs to be configured.

**Comment by mpreisler** [comment]
ACK

**Pull request merged**